boofuzz/utils/debugger_thread_pydbg.py

```diff
@@ -162,6 +162,9 @@ def _scan_proc_names_once(self):
     def stop_target(self):
         try:
             os.system("taskkill /pid %d" % self.pid)
```

Comment:
Why need to use `taskkill` twice?

Reply:
the key is the `-f` in the second `taskkill` call -- the idea is to forcefully kill the process if necessary. https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/taskkill

Reply:
Yes, I know the `-F` option. Actually, I mean the two `taskkill` before `taskkill -F`. By the way, maybe using `taskkill /F` instead of `taskkill -F` to keep consistent?
https://github.com/jtpereyda/boofuzz/blob/45eef0c3d3585fd027e4a693d18e53d1a1916006/boofuzz/utils/debugger_thread_pydbg.py#L164-L167

Reply:
Ah, so the thinking is the initial taskkill will send the more typical SIGKILL and give the application a chance to more gracefully shut down, kind of like trying to close an application before breaking out Task Manager.

Reply:
Yeah, `/F` would be more consistent with what's already there.

Reply:
Yes.  What I mean is to do it like this, without the first statement `os.system("taskkill /pid %d" % self.pid)`. Otherwise, the three `os.system()` will be executed in most cases.

    exit_code = os.system("taskkill /pid %d" % self.pid) 
    if exit_code != 0: 
        os.system("taskkill /F /pid %d" % self.pid)

Reply:
🤦‍♂ oh no, I was totally not getting it. XD Yep that is right I will need to open another PR

Reply:
ok.

Reply:
Feel free to push the fix if you're in a hurry, but it's my mess so I'll try to clean it up. Thanks for sticking with me until I got it. :)

Reply:
I just want to make sure I'm not understood. Also, I just do some common code review. So you can do it when you're free.